ci/test_wheel_pylibwholegraph.sh

```diff
@@ -23,7 +23,7 @@ mkdir -p "${RAPIDS_TESTS_DIR}" "${RAPIDS_COVERAGE_DIR}"
 
 # echo to expand wildcard before adding `[extra]` requires for pip
 rapids-logger "Installing Packages"
-rapids-retry python -m pip install \
+rapids-retry rapids-pip-retry install \
```

Comment:
We probably want this:
```suggestion
rapids-pip-retry install \
```